Procedures&Validations/upgrade-procedure.md

```diff
@@ -0,0 +1,58 @@
+## High Level Hard Fork (HF) Procedure
+
+1. Determine suitable version of the clients for OE / Nethermind (3.3.5 for OE, 1.31.12 for Nethermind).
```

Comment:
What if we explicitly write the full standing for `OE` once here and then leave OE in the rest of the doc ?

```suggestion
1. Determine suitable version of the clients for OpenEthereum (OE) / Nethermind (3.3.5 for OE, 1.31.12 for Nethermind).
```

Reply:
This line might confuse external parties. 
For OpenEthereum it is always version `v3.3.5`; 
but for Nethermind It is `1.31.12` for Volta network and we don't know yet for EWC network. I assume it might be `1.31.13`. This ll include both Volta and EWC network.
Still we can't say also Nethermind `1.31.13` or higher as Nethermind already released `1.32.2` and these release doesn't upgraded for Volta/EWC (chainspec.)

Reply:
Will modify that